### PR TITLE
Updating root README

### PR DESCRIPTION
**Pull request by WillieHabi**
`npm run start` does not start local server for any of the examples. Must run npm start:server first to start local azure service/tinylicious

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,14 @@ To get started, clone the repository and navigate to one of the example folders.
 From the example root, you can install and start the example.
 
 1. Run `npm install` from the example root
-2. Run `npm run start` to start both the client and server
+2. Run `npm start:server` to start the local server
+3. Run `npm start` to start the client
```
Comment (scottn12) [inline README.md:16]: Which example does this actually run? We should probably specify that in the README too

Comment (WillieHabi) [reply README.md:16]: I think these instructions work with every example in the repo, if I'm not mistaken

Comment (scottn12) [reply README.md]: Ah gotcha! That's on me for misunderstanding 😅

**Comment by ChumpChief** [comment]
Node-demo has since been removed from the repo, so closing this PR.

**Pull request closed without merging**